packages/foreman/foreman/foreman.spec

```diff
@@ -939,12 +940,14 @@ rm -rf %{buildroot}
 %ghost %{_datadir}/%{name}/config/initializers/encryption_key.rb
 %ghost %attr(0640,root,%{name}) %config(noreplace) %{_datadir}/%{name}/config/initializers/local_secret_token.rb
 %{_tmpfilesdir}/%{name}.conf
+%{_sysusersdir}/%{name}.conf
 
 %pre
-# Add the "foreman" user and group
-getent group %{name} >/dev/null || groupadd -r %{name}
-getent passwd %{name} >/dev/null || \
-useradd -r -g %{name} -d %{homedir} -s /sbin/nologin -c "Foreman" %{name}
+%if 0%{rhel} < 9
+%sysusers_create_compat extras/systemd/%{name}.sysusers
+%else
+%sysusers_create_package %{name} extras/systemd/%{name}.sysusers
```

Comment:
`%sysusers_create_compat` isn't in EL8 and we don't want to depend on `epel-rpm-macros` (I think). Same here that I'm not sure if the last path is valid given it isn't documented.